ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/MachineAccessIntegrationForm.tsx

```diff
@@ -0,0 +1,377 @@
+                    <FormikProvider value={formik}>
+                        <FormLabelGroup
+                            isRequired
+                            label="Select configuration type"
+                            fieldId="type"
+                            touched={touched}
+                            errors={errors}
+                        >
+                            <SelectSingle
+                                id="type"
+                                value={values.type}
+                                handleSelect={(name, value) => {
+                                    setFieldValue(name, value);
+                                    setFieldValue(
+                                        'issuer',
+                                        value === 'GITHUB_ACTIONS'
+                                            ? 'https://token.actions.githubusercontent.com'
+                                            : ''
+                                    );
+                                }}
+                                direction="down"
+                                isDisabled={!isEditable}
+                            >
+                                <SelectOption key={'GENERIC'} value={'GENERIC'}>
+                                    Generic
+                                </SelectOption>
+                                <SelectOption key={'GITHUB_ACTIONS'} value={'GITHUB_ACTIONS'}>
+                                    Github action
+                                </SelectOption>
+                            </SelectSingle>
+                        </FormLabelGroup>
+                        <FormLabelGroup
+                            isRequired
+                            label="Issuer"
+                            fieldId="issuer"
+                            touched={touched}
+                            errors={errors}
+                        >
+                            <TextInput
+                                isRequired
+                                type="text"
+                                id="issuer"
+                                value={values.issuer}
+                                onChange={onChange}
+                                onBlur={handleBlur}
+                                isDisabled={!isEditable || values.type === 'GITHUB_ACTIONS'}
+                            />
+                        </FormLabelGroup>
+                        <FormLabelGroup
+                            isRequired
+                            label="Token lifetime"
+                            fieldId="tokenExpirationDuration"
+                            touched={touched}
+                            errors={errors}
+                        >
+                            <TextInput
+                                isRequired
+                                type="text"
+                                id="tokenExpirationDuration"
+                                placeholder={'3h20m20s'}
```

Comment:
Placeholders have fallen out of favor, because they can easily be mistaken for already-entered values, they disappear as soon as the user starts typing, and they can be confusing for users with accessibility needs.

The UI team has moved to using the PatternFly component's `helperText` prop to display always-visible examples, as in [the Team integration form](https://github.com/stackrox/stackrox/blob/0a7614a0d60478d568cdabd0f0b2e7e507db5124/ui/apps/platform/src/services/AdministrationEventsService.ts#L40-L60).

```suggestion
                        <FormLabelGroup
                            isRequired
                            label="Token lifetime"
                            fieldId="tokenExpirationDuration"
                            touched={touched}
                            errors={errors}
                            helperText="For example, 3h20m20s"
                        >
                            <TextInput
                                isRequired
                                type="text"
                                id="tokenExpirationDuration"
```